cursorless-talon/src/snippets.py

```diff
@@ -90,14 +90,20 @@ def insert_named_snippet(
     insert_snippet(snippet, destination)
 
 
-def insert_custom_snippet(body: str, destination: CursorlessDestination):
-    insert_snippet(
-        {
-            "type": "custom",
-            "body": body,
-        },
-        destination,
-    )
+def insert_custom_snippet(
+    body: str,
+    destination: CursorlessDestination,
+    scope_types: Optional[list[str]] = None,
```

Comment:
I think this function should accept a list of scope types, rather than scope type types. The latter is just a convenience for our public api